src/services/server/LoggedInState.cpp

```diff
@@ -67,7 +67,7 @@ LoggedInState::~LoggedInState()
 void LoggedInState::checkTransfer()
 {
 	TransferInfoStringSerializer serializer;
-	std::string teleportFilePath(EmberServices::getSingleton().getConfigService().getHomeDirectory() + "/teleports");
+	std::string teleportFilePath(EmberServices::getSingleton().getConfigService().getHomeDirectory(BaseDirType_CACHE) + "teleports");
```

Comment:
This shouldn't be in cache.
The transfer information is crucial for letting a player follow their avatar as it moves between servers. Think of it like a password, or a key. If it's lost the avatar is lost also.